server/game/cards/01_SOR/units/SeventhSisterImplacableInquisitor.ts

```diff
@@ -0,0 +1,34 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { Location, RelativePlayer, WildcardCardType } from '../../../core/Constants';
+import { DamageType } from '../../../core/Constants';
+
+
+export default class SeventhSisterImplacableInquisitor extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '0828695133',
+            internalName: 'seventh-sister#implacable-inquisitor',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addTriggeredAbility({
+            title: 'When this unit deals combat damage to an opponent’s base: You may deal 3 damage to a ground unit that opponent controls',
+            when: {
+                onDamageDealt: (event, _context) =>
```

Comment:
Can you do me a small favor and add a comment here that just says: `TODO: refactor damage enum types to account for the fact that overwhelm is combat damage`